stages/org.osbuild.oscap.remediation

```diff
@@ -35,7 +35,7 @@ def main(tree, options):
     for source in ("/dev", "/proc"):
         target = os.path.join(tree, source.lstrip("/"))
         os.makedirs(target, exist_ok=True)
-        mount(source, target, ro=False)
+        mnt.mount(source, target, ro=False)
```

Comment:
This change (and the added test) are not required for this PR, is that right? It looks orthogonal and maybe could be it's own PR so that both are easier to review as they don't mix concerns?